core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -24,7 +24,10 @@
  */
 
 public final class TenantConstants extends RequestResponseApiConstants {
-    /* message id prefix */
+
+    /**
+     *  All messages for the Tenant API will start with this prefix.
```

Comment:
Let's hope that it's only the message's IDs that start with this prefix ...

Reply:
Like the slight sarcasm :+1: 
Will change it to 
`Messages that are sent by the Hono client for the Tenant API use this as a prefix for the messageId.`
IMHO this improves the description...